core/src/CoreTelemetryService.cs

```diff
@@ -47,5 +42,14 @@ public static ITelemetry GetInstance()
         {
             return instance;
         }
+
+        public static TelemetryHelper CreateTelemetryHelper(
+            string requestId,
+            EventBase eventToStart,
+            EventBase eventToEnd = null,
+            bool shouldFlush = false)
+        {
+            return new TelemetryHelper(GetInstance(), requestId, eventToStart, eventToEnd ?? eventToStart, shouldFlush);
+        }
```

Comment:
I understand why it's here now, but... I'm having some debates with myself... does this belong here?

Reply:
As a stepping stone, I'd say it belongs here.  I also updated other parts of the code to all route into CoreTelemetryService as the central choke point since some were going to Telemetry.GetInstance() as well.  it looks like CoreTelemetryService exists so that we can override the value of the ITelemetry global instance, since you can't set it on the Telemetry class directly.  I'd like to look at our global object factory/singleton mechanisms more broadly in later work.